_pages/robots.html

```diff
@@ -11,10 +11,15 @@
     <!-- TODO: Expand this page. There's not enough content. Also, would be nice to have a more interesting layout. maybe even have pixel drawings of the robots? -->
     <h1>The Robots</h1>
     <div>
+        <div>
+            <img src="/images/robots/2022.jpg">
+            <h5>2022</h5>
+            <p>Our 2022 robot, Hippo Esurientem, was built for the <a href="https://www.youtube.com/watch?v=LgniEjI9cCM"> FIRST RAPID REACT competition</a>. It has a retractable intake arm that can pick up cargo (large tennis balls) and a launcher that shoot them either high or low, by using a motor to spin wheels at thousands of RPM. The robot also sports an extendable climbing hook which enables the robot to get off the ground at the end of the match. <a href="/robot/2022">More about our 2022 robot</a></p>
+        </div>
         <div>
             <img src="/images/robots/2020.png">
             <h5>2020</h5>
-            <p>Our 2020 robot, Mouse, is being built for the <a href="https://www.youtube.com/watch?v=gmiYWTmFRVE"> FIRST INFINITE RECHARGE competition</a>. It has an intake system that can take up powercells (softballs) and shoot them at varying distances, by using a motor to spin wheels very fast, into a tall tower. It also has a wheel that extends by a piston that is used to spin the control panel (color wheel) with various sensors to detect the color and the closeness of the structure to the robot. Finally, we have installed a scissor lift that extends to hook onto the seesaw and a winch that is used for lifting the robot off the ground during matches. <a href="/robot/2020">More about our 2020 robot</a></p>
+            <p>Our 2020 robot, Mouse, was built for the <a href="https://www.youtube.com/watch?v=gmiYWTmFRVE"> FIRST INFINITE RECHARGE competition</a>. It has an intake system that can take up powercells (softballs) and shoot them at varying distances, by using a motor to spin wheels very fast, into a tall tower. It also has a wheel that extends by a piston that is used to spin the control panel (color wheel) with various sensors to detect the color and the closeness of the structure to the robot. Finally, we have installed a scissor lift that extends to hook onto the seesaw and a winch that is used for lifting the robot off the ground during matches. <a href="/robot/2020">More about our 2020 robot</a></p>
```

Comment:
Nit: there's an extra space at the start of the first link

Reply:
I'm sorry, I'm not sure where the extra space is. Can you elaborate?

Reply:
I just saw it, and have removed the extra spaces :)